Android/KotlinXCoroutines/source/KotlinX.Coroutines.Android/KotlinX.Coroutines.Android.csproj

```diff
@@ -39,8 +43,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\..\externals\kotlinx-coroutines-android-$(PackageVersion).jar" Pack="True" PackagePath="jar\kotlinx-coroutines-android-$(PackageVersion).jar" Visible="false" />
-    <EmbeddedJar Include="..\..\externals\kotlinx-coroutines-android-$(PackageVersion).jar" Link="Jars\kotlinx-coroutines-android-$(PackageVersion).jar" />
+    <None Include="..\..\externals\kotlinx-coroutines-android-$(ArtifactVersion).jar" Pack="True" PackagePath="jar\kotlinx-coroutines-android-$(ArtifactVersion).jar" Visible="false" />
+    <EmbeddedJar Include="..\..\externals\kotlinx-coroutines-android-$(ArtifactVersion).jar" Link="Jars\kotlinx-coroutines-android-$(ArtifactVersion).jar" />
```

Comment:
What do these lines do?

It seems like they're going to add the `.jar` to the `.nupkg` twice: once embedded in the `.dll` and once as a loose `.jar` file.

Reply:
I think this is for `buildTransitive` (but I am no expert on this)

https://github.com/xamarin/XamarinComponents/issues/1136

and

https://github.com/xamarin/XamarinComponents/issues/1017

Reply:
It looks to be added twice, but this is probably because the accompanying .targets file is not added to the nuget - and then this was added back.

Reply:
It seems like we shouldn't be adding `.targets` files or adding loose `.jar` files at all, but I guess we do it with all of our packages so I guess it's ok.